pkg/webui/console/views/application-api-keys-list/index.js

```diff
@@ -21,7 +21,7 @@ import ApiKeysTable from '../../containers/api-keys-table'
 import { getApplicationApiKeysList } from '../../store/actions/applications'
 import sharedMessages from '../../../lib/shared-messages'
 
-const API_KEYS_TABLE_SIZE = 10
+import pageSizes from '../../constants/page-sizes'
```

Comment:
```suggestion
import PAGE_SIZES from '../../constants/page-sizes'
```

We usually use UPPER_CASE for constant object imports. Having that in mind, it actually makes more sense (and is more consistent) to rename the constant values to allow for a usage like `PAGE_SIZES.REGULAR`